mne/surface.py

```diff
@@ -229,6 +256,54 @@ def get_meg_helmet_surf(info, trans=None, verbose=None):
     return surf
 
 
+def _scale_helmet_to_sensors(system, surf, info):
+    fname = _helmet_path / f"{system}_ch_pos.txt"
+    if not fname.is_file():
+        return surf
+    with open(fname) as fid:
+        ch_pos_from = json.load(fid)
+    # find correspondence
+    fro, to = list(), list()
+    for key, f_ in ch_pos_from.items():
+        t_ = [ch["loc"][:3] for ch in info["chs"] if ch["ch_name"].startswith(key)]
+        if not len(t_):
+            continue
+        fro.append(f_)
+        to.append(np.mean(t_, axis=0))
+    if len(fro) < 4:
+        logger.info(
+            "Using untransformed helmet, not enough sensors found to deform to match "
+            f"acquisition based on sensor positions (got {len(fro)}, need at least 4)"
+        )
```

Comment:
@drammock took a stab at improving these, see if you think they make more sense now

Reply:
that works, thanks!